e2e/DETOX_DOC.md

```diff
@@ -0,0 +1,30 @@
+# End to end testing with Detox
+
+## iOS setup instructions
+
+1. Copy code from readme into info.plist to get the ios simulator working ([instructions](https://github.com/cds-snc/covid-alert-app#ios-local-development)).
+2. `yarn install`, `sudo gem install cocoapods`, `bundle install && yarn pod-install`
+3. `brew tap wix/brew && brew install applesimutils`
+4. `yarn pre:test:ios`
```

Comment:
Might want to break this out into running the tests as this part will happen over and over.